src/App.vue

```diff
@@ -875,6 +873,8 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // set the resources
     if (!resources) throw new Error(`Invalid ${this.getEntityType} resources`)
     this.setResources(resources)
+    // NB - for some reason, need to call this here so the store updates this getter
+    const dummy = this.getFilingData // eslint-disable-line
```

Comment:
This is very strange. Without this here, `getFilingData` always returned null. I tried a few things, including awaiting next tick, refactoring getFilingData, refactoring the failing App unit test, etc, and this is the only thing that worked consistently.

In any case, this code is safe (won't hurt anything), but I am a tad concerned about other getters that don't work after just setting a value/object in the store.